### PR TITLE
feat(bigtable): enable keepalive pings by default

### LINKED ISSUE #5662: bigtable: update keepalive values
Update bigtable keepalive values:
keepalive time: 30s
keepalive timeout: 10s

Example Java reference:
https://github.com/googleapis/java-bigtable/pull/528

### PR DESCRIPTION
**Pull request by coryan**
This ensures dropped connections are detected promptly.

Fixes #5662

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/5969)

### CODE REVIEW

File: google/cloud/bigtable/client_options.cc [hunk 1]
```diff
@@ -118,12 +133,31 @@ ClientOptions::ClientOptions(std::shared_ptr<grpc::ChannelCredentials> creds)
...
       BIGTABLE_CLIENT_DEFAULT_MAX_MESSAGE_LENGTH);
   channel_arguments_.SetMaxReceiveMessageSize(
       BIGTABLE_CLIENT_DEFAULT_MAX_MESSAGE_LENGTH);
+  auto to_arg = [](std::chrono::milliseconds ms) {
+    auto const count = ms.count();
+    if (count >= (std::numeric_limits<int>::max)()) {
+      return std::numeric_limits<int>::max();
+    }
+    if (count <= (std::numeric_limits<int>::min)()) {
+      return std::numeric_limits<int>::max();
```
Comment (devbww) [inline google/cloud/bigtable/client_options.cc:151]: s/max/min/?

File: google/cloud/bigtable/client_options_test.cc [hunk 2]
```diff
@@ -35,6 +35,32 @@ namespace {
 
 using ::testing::HasSubstr;
 
+absl::optional<int> GetInt(grpc::ChannelArguments const& args,
+                           std::string const& key) {
+  auto c_args = args.c_channel_args();
+  // Just do a linear search for the key, the data structure is not organized
```
Comment (devbww) [inline google/cloud/bigtable/client_options_test.cc:41]: s/,/;/ perhaps.

File: google/cloud/bigtable/client_options_test.cc [hunk 3]
```diff
@@ -35,6 +35,32 @@ namespace {
...
+    if (a->type != GRPC_ARG_INTEGER) return absl::nullopt;
+    return a->value.integer;
+  }
+  return absl::nullopt;
+}
+
+absl::optional<std::string> GetString(grpc::ChannelArguments const& args,
+                                      std::string const& key) {
+  auto c_args = args.c_channel_args();
+  // Just do a linear search for the key, the data structure is not organized
```
Comment (devbww) [inline google/cloud/bigtable/client_options_test.cc:54]: Ditto.

**Review by devbww — APPROVED** [review]

File: google/cloud/bigtable/client_options.cc (hunk 1 above)
Comment (coryan) [reply google/cloud/bigtable/client_options.cc]: Fixed, thanks.

File: google/cloud/bigtable/client_options_test.cc (hunk 2 above)
Comment (coryan) [reply google/cloud/bigtable/client_options_test.cc]: Done.

File: google/cloud/bigtable/client_options_test.cc (hunk 3 above)
Comment (coryan) [reply google/cloud/bigtable/client_options_test.cc]: Done.

**Pull request merged**